### PR TITLE
Fix recovery of kdump_and_crash test after soft-fail

### PR DESCRIPTION
**Pull request by mimi1vx**
Original after successful application of workaround caused early end of test.
Now test continue with crash and its analyze

http://quasar.suse.cz/tests/102#

### CODE REVIEW

File: lib/kdump_utils.pm [hunk 1]
```diff
@@ -110,7 +110,7 @@ sub kdump_is_active {
             record_soft_failure 'bsc#1021484 -- fail to create kdump initrd';
             assert_script_run 'systemctl restart kdump';
             $status = script_output('systemctl status kdump ||:');
```
Comment (rwx788) [inline lib/kdump_utils.pm]: I guess we can get rid of this line also then. I'm also not sure if we want to see 10 soft-failures, but just one, and keep restarting it.

**Comment by okurz** [comment]
In subject s/after/after

**Comment by okurz** [comment]
I understand the subject line but I fail to understand the description in the commit, sorry

File: lib/kdump_utils.pm (hunk 1 above)
Comment (mimi1vx) [reply lib/kdump_utils.pm]: @rwx788 not possible. 

```assert_script_run 'systemctl restart kdump';``` - causes hard fail of test if restart of service fails

**Comment by okurz** [comment]
LGTM

**Pull request merged**